assets/dev/scss/frontend/_container.scss

```diff
@@ -6,6 +6,11 @@
 	--flex-basis: auto;
 	--flex-grow: 0;
 	--flex-shrink: 1;
+	--container-widget-width: 100%;
+	--container-widget-height: initial;
+	--container-widget-flex-grow: 0;
+	--container-widget-align-self: initial;
+
```

Comment:
Without specifying these values in the SCSS, child containers inherited these styles from parent containers.